src/test/java/io/knowledgeassets/myskills/server/common/AbstractControllerTests.java

```diff
@@ -24,7 +25,7 @@
  * </ul>
  * <p>Provides convenient builder methods to configure user permissions checked by method security.</p>
  */
-@Import({Neo4jSessionFactoryConfiguration.class, MethodSecurityConfiguration.class})
+@Import({Neo4jSessionFactoryConfiguration.class, MethodSecurityConfiguration.class, SecurityService.class})
```

Comment:
Should we maybe rather mock the `SecurityService` itself instead of mocking all of its dependencies (currently only `CommunityQueryService`)? I think it can get cumbersome to mock all future security service dependencies in each and every controller test.

Reply:
Thanks. Done.